### PR TITLE
Fix kops aws jobs to follow stableN pattern

### PR DESCRIPTION
**Pull request by justinsb**
Also fixed two jobs with the same cluster name, which I imagine was
causing problems.

### CODE REVIEW

File: jobs/config.json [hunk 1]
```diff
@@ -8498,6 +8497,22 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-aws-latest": {
```
Comment (krzyzacy) [inline jobs/config.json]: this is same as ci-kubernetes-e2e-kops-aws?

**Comment by krzyzacy** [comment]
and also remove ci-kubernetes-e2e-kops-aws-release-1-7

thanks for the bikeshed :-)

File: jobs/config.json (hunk 1 above)
Comment (justinsb) [reply jobs/config.json]: It is - good spot.  I'll remove it.  I need to do a pass on all these jobs to make sure we don't have dups (and maybe name them following a pattern).  And I also need to move this job to the new runner (almost there!)

**Comment by justinsb** [comment]
Fixed up and replaced the 1.7 job with stable3.

Thanks for splitting out the critical problem with the name of the 1.9 job though :-)

**Comment by krzyzacy** [comment]
/lgtm
sounds like we can get rid of the kops-aws-runner.sh soon ish :-)

**Pull request merged**